tests/BasicSearchTests.FunctionalTests.Core/EnvironmentSettings.cs

```diff
@@ -47,6 +50,45 @@ public static string IndexBaseUrl
             }
         }
 
+        public static string AzureSearchIndexUrl
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(_azureSearchIndexUrl))
+                {
+                    _azureSearchIndexUrl = GetEnvironmentVariable("AzureSearchIndexUrl", "https://nuget-int-usnc.search.windows.net");
+                }
+
+                return _azureSearchIndexUrl;
+            }
+        }
+
+        public static string AzureSearchIndexApiKey
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(_azureSearchIndexApiKey))
+                {
+                    _azureSearchIndexApiKey = GetEnvironmentVariable("AzureSearchIndexApiKey", "");
+                }
+
+                return _azureSearchIndexApiKey;
+            }
+        }
+
+        public static string AzureSearchIndexName
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(_azureSearchIndexName))
+                {
+                    _azureSearchIndexName = GetEnvironmentVariable("AzureSearchIndexName", "search");
```

Comment:
throw?

Reply:
Talked offline. This is the pattern used by other settings, I will keep this setting consistent.